idmapping-rest/src/test/java/org/uniprot/api/idmapping/controller/utils/IdMappingUniProtKBITUtils.java

```diff
@@ -98,6 +98,18 @@ public static String getUniProtKbFieldValueForValidatedField(String searchField)
                 case "annotation_score":
                     value = "5";
                     break;
+                case "uniref_cluster_50":
+                    value = "UniRef50_P0001";
+                    break;
+                case "uniref_cluster_90":
+                    value = "UniRef90_P0001";
+                    break;
+                case "uniref_cluster_100":
+                    value = "UniRef100_P0001";
+                    break;
```

Comment:
I think the accession part should have another char, perhaps. 
UniRef50_P0001 -> UniRef50_P00001 ?

It probably makes no difference tho!